cmd/sovereignnode/chainSimulator/tests/epochChange/epochChange_test.go

```diff
@@ -99,6 +99,7 @@ func TestSovereignChainSimulator_EpochChange(t *testing.T) {
 					},
 				}
 
+				newCfg.AndromedaEnableEpoch = 2
```

Comment:
If we keep the flag, maybe run this tests 2 times, with andromeda epoch 0 and 2

Reply:
Other tests work with this flag on 0.
I changed this here to have a test working with transition to Andromeda